tests/driver_at86rf2xx/cmd.c

```diff
@@ -301,6 +301,28 @@ static int send(int iface, le_uint16_t dst_pan, uint8_t *dst, size_t dst_len,
         puts(")");
     }
     return 0;
+
+}
+void random_net_api(uint8_t idx, uint32_t *value)
+{
+    netdev_ieee802154_t *dev = &devs[idx].netdev;
+    int (*get)(netdev_t *, netopt_t, void *, size_t) = dev->netdev.driver->get;
+    get(&dev->netdev, NETOPT_RANDOM, value, sizeof(uint32_t));
 }
 
+int random(int argc, char **argv)
+{
+    (void) argc;
+    (void) argv;
```

Comment:
```suggestion
    (void)argc;
    (void)argv;
```

I personally prefer it to have casts formatted like unary operators.